assets/src/settings-page/welcome.js

```diff
@@ -76,19 +76,20 @@ export function Welcome() {
 					</h2>
 					<p>
 						{ __( 'The AMP configuration wizard helps you choose the best configuration settings for your site.', 'amp' ) }
+						{ ' ' }
+						<a href={ onboardingWizardLink } >
+							{ pluginConfigured ? __( 'Reopen Wizard', 'amp' ) : __( 'Open Wizard', 'amp' ) }
+						</a>
+						{ customizerLink && templateModeWasOverridden === false && (
+							<>
+								{ ` ${ _x( 'or', 'e.g. do this or that', 'amp' ) } ` }
+								<a href={ customizerLink } >
+									{ __( 'Customize Reader Theme', 'amp' ) }
+								</a>
+							</>
+						) }
+						{ '.' }
```

Comment:
This punctuation may need to be translatable. See https://qr.ae/pGwidt 

```suggestion
						{ _x( '.', 'End of sentence.', 'amp' ) }
```